integration/core/test_cli.py

```diff
@@ -122,6 +122,42 @@ def test_replica_add_start(bin, grpc_controller_client,  # NOQA
     assert volume.replicaCount == 1
 
 
+def check_rebuild_status(bin, url):
```

Comment:
Rename this function since it is actually waiting for rebuild complete.

Reply:
Sure, will use ->`wait_and_fetch_rebuild_status`